contributions/presentation/week5/vladom-johanmb/README.md

```diff
@@ -0,0 +1,18 @@
+# Presentation
+
+
+ ## Members
+ Vlado Mitrovic (vladom@kth.se)
+ Github username: [vladomitrovic](https://github.com/vladomitrovic)
+
+ Johan Mallo Bakken (johanmb@kth.se)
+ Github username: [johanmallobakken](https://github.com/johanmallobakken)
+ 
+ ## Topic
+ 
+The goal is to present different agile methodologies approaches (scrum, kanban, scrumban, ...) and how they are related to Devops.
+
+- Introduction to agile methods
+- Scrum vs Kanban vs Scrumban
+- Tools examples (JIRA, Azure DevOps Boards, Tuleap, ...)
```

Comment:
Do you want to explain how these tools are used to implement agile methodologies or are these tools a whole different topic?

Reply:
Please make this point clear in your proposal.

Reply:
Done